src/translations/en.json

```diff
@@ -1706,6 +1706,14 @@
         "update": "[%key:ui::panel::config::devices::update%]",
         "unknown_error": "[%key:ui::panel::config::devices::unknown_error%]"
       },
+      "label-detail": {
+        "new_label": "New label",
+        "name": "Name",
+        "icon": "Icon",
+        "color": "Color",
+        "description": "Description",
+        "required_error_msg": "[%key:ui::panel::config::zone::detail::required_error_msg%]"
```

Comment:
Not great that we use the zone config text here but I see we already use it in a bunch of places :(
Have move that `required_error_msg` to a common place